### ISSUE TITLE
Add city table to API

### ISSUE THREAD

**Issue by colindavey** [OpenEugene/little-help-book-api#4]
Make a CitiesController.cs in LittleHelpBook.Server, and follow the pattern from another controller to read the data and return JSON.

**Comment by markdav-is** [comment]
I think this was addressed in a PR!

**Comment by markdav-is** [comment]
@colindavey  we still need to deploy this right?

**Comment by colindavey** [comment]
Yes, but first I need to get one more table and field implemented.

**Comment by markdav-is** [comment]
you can deploy with every PR and test it to make sure it's working.  no reason to wait.  feature branches and issues should be atomic.  The workflow is open-issue->feature branch->PR->merge->deploy->test->close-issue.  We also have dev-, test- and prod- mvp-studio environments.  So we can also managing merging dev to main ala "gitflow"

**Comment by colindavey** [comment]
I just did a _draft_ PR to get feedback from you that I'm barking up the right tree. I'm still testing it and finding issues running locally. I'm shooting for a PR when I can successfully use the Swagger API locally. It's a process of updating the front end and the API in sync.

**Comment by markdav-is** [comment]
we can try deploying to dev so you can test it online.
